README-pt-BR.md

```diff
@@ -0,0 +1,347 @@
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![Todos os contribuidores](https://img.shields.io/badge/todos_contribuidores-175-orange.svg)](#contributors)
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
+
+| Teste | Status |
+| :--- | :--- |
+| Versão Netlify | [![CI](https://github.com/alan-turing-institute/the-turing-way/workflows/CI/badge.svg)](https://github.com/alan-turing-institute/the-turing-way/actions?query=workflow%3ACI+branch%3Amaster) |
+| Sem Frases Latinas | [![Check for Latin Phrases](https://github.com/alan-turing-institute/the-turing-way/workflows/Check%20for%20Latin%20Phrases/badge.svg)](https://github.com/alan-turing-institute/the-turing-way/actions?query=workflow%3A%22Check+for+Latin+Phrases%22+branch%3Amaster) |
+| Sem Grandes Arquivos | [![TestFileSizes](https://github.com/alan-turing-institute/the-turing-way/workflows/TestFileSizes/badge.svg)](https://github.com/alan-turing-institute/the-turing-way/actions?query=workflow%3ATestFileSizes+branch%3Amaster) |
+| Sem "_Lorem Ipsum_"s | [![Check for Lorem Ipsums](https://github.com/alan-turing-institute/the-turing-way/workflows/Check%20for%20Lorem%20Ipsums/badge.svg)](https://github.com/alan-turing-institute/the-turing-way/actions?query=workflow%3A%22Check+for+Lorem+Ipsums%22+branch%3Amaster) |
+
+O _The Turing Way_ é um guia levemente obstinado para a ciência de dados reproduzível.
+Você pode lê-lo aqui: <https://the-turing-way.netlify.com>
+Neste momento, você está visualizando o repositório GitHub do projeto onde se encontra cada pedacinho que compõe o guia e onde o processo de escrita/construção do guia acontece.
+
+Nosso objetivo é prover todas as informações que os pesquisadores precisam no início de seus projetos para assegurar que eles serão facilmente reproduzíveis quando terminados.
+
+Isso também significa garantir que os alunos de doutorado, pós-doutorandos, pesquisadores principais e equipes de financiamento saibam quais partes da "responsabilidade de reprodutibilidade" podem afetar e o que devem fazer para estimular a ciência de dados a ser mais eficiente, eficaz e compreensível.
```

Comment:
Isso também significa garantir que os alunos de doutorado, pós-doutorandos, pesquisadores principais e equipes de financiamento saibam quais partes da "responsabilidade de reprodutibilidade" eles podem afetar, e o que devem fazer para estimular a ciência de dados a ser mais eficiente, eficaz e compreensível.

Reply:
It does sound better!